performance/testFactory.ts

```diff
@@ -3,6 +3,7 @@ import {ExtrinsicTransfer} from "./plugins/extrinsicTransfer";
 import {TestItem} from "./plugins/testItem";
 
 export enum Tests {
+  Undefined,
```

Comment:
We should represent this in the type system with `Tests | undefined` rather than as an additional enum